docs/admin/migration.md

```diff
@@ -0,0 +1,12 @@
+# Migrating from KFServing
+
+This doc helps to migrate inference services from `KFServing` to `KServe` in
+near zero downtime. The script installs KServe from latest release version.
+
+
+### Migration
+
+```bash
+    cd hack
+    ./kserve_migration.sh
```

Comment:
This is the mode you can run outside of the kube cluster which also requires the permission to all namespaces, I think the better way is what we have discussed to run this as an kubernetes job in `kserve` namespace as that service acount has the proper rbac to operate inference services at cluster scope.